packages/Button/src/LinkButton.js

```diff
@@ -17,7 +18,8 @@ const LinkPropTypes = {
 
 const LinkDefaultProps = {
   a11yText: null,
-  isOpenNewTab: true,
+  isDisabled: false,
+  isOpenNewTab: false,
```

Comment:
Maybe if I'm introducing a breaking change already, I should go all in and rename the prop to `shouldOpenNewTab`??

Reply:
@mikrotron ha, I was going to suggest the exact same thing